src/main/java/net/dries007/tfc/common/blockentities/BloomeryBlockEntity.java

```diff
@@ -258,13 +259,13 @@ private void dumpItems()
     {
         assert level != null;
 
-        BlockPos pos = getExternalBlock();
-        int x = pos.getX();
-        int y = pos.getY();
-        int z = pos.getZ();
-        inputStacks.forEach(i -> Containers.dropItemStack(level, x, y, z, i));
-        catalystStacks.forEach(i -> Containers.dropItemStack(level, x, y, z, i));
+        final BlockPos pos = getExternalBlock();
+        for (ItemStack stack : inputStacks)
+        {
+            Containers.dropItemStack(level, pos.getX(), pos.getY(), pos.getZ(), stack);
```

Comment:
Any reason we are using this over `Helpers#spawnItem`? I know it basically doesn't matter